### DISCUSSION TITLE
1.0.8

### DISCUSSION THREAD

**Discussion by Schwartzblat** [Schwartzblat/WhatsApp-Web-Plus#34] [Announcements]
- Improve the mechanic of tagging everyone and admins

<hr /><em>This discussion was created from the release <a href='https://github.com/Schwartzblat/WhatsApp-Web-Plus/releases/tag/1.0.8'>1.0.8</a>.</em>

**Comment by melonflari** [comment]
Does the extension allow you to give yourself administrator rights?

**Reply by Schwartzblat** [reply]
Nope

**Reply by melonflari** [reply]
Could you please tell me how to create an extension that, when installed, will allow you to give yourself administrator rights in any WhatsApp group? I've been trying to write such an extension for a week, but my knowledge is not enough
